src/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -195,239 +110,123 @@ public Bitmap(Stream stream, bool useIcm)
-            }
+            ValidateBitmap(bitmap);
 
             SetNativeImage(bitmap);
-
             EnsureSave(this, null, stream);
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.Bitmap5"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the
-        ///       Bitmap class with the specified size, pixel format, and pixel data.
-        ///    </para>
-        /// </devdoc>
         public Bitmap(int width, int height, int stride, PixelFormat format, IntPtr scan0)
         {
             IntPtr bitmap = IntPtr.Zero;
-
             int status = SafeNativeMethods.Gdip.GdipCreateBitmapFromScan0(width, height, stride, unchecked((int)format), new HandleRef(null, scan0), out bitmap);
-
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
+            SafeNativeMethods.Gdip.CheckStatus(status);
 
             SetNativeImage(bitmap);
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.Bitmap6"]/*' />
-        /// <devdoc>
-        ///    <para>
-        ///       Initializes a new instance of the Bitmap class with the specified
-        ///       size and format.
-        ///    </para>
-        /// </devdoc>
         public Bitmap(int width, int height, PixelFormat format)
         {
             IntPtr bitmap = IntPtr.Zero;
-
             int status = SafeNativeMethods.Gdip.GdipCreateBitmapFromScan0(width, height, 0, unchecked((int)format), NativeMethods.NullHandleRef, out bitmap);
-
-            if (status != SafeNativeMethods.Gdip.Ok)
-                throw SafeNativeMethods.Gdip.StatusException(status);
+            SafeNativeMethods.Gdip.CheckStatus(status);
 
             SetNativeImage(bitmap);
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.Bitmap7"]/*' />
-        /// <devdoc>
-        ///    Initializes a new instance of the
-        /// <see cref='System.Drawing.Bitmap'/> 
-        /// class with the specified size.
-        /// </devdoc>
-        public Bitmap(int width, int height) : this(width, height, System.Drawing.Imaging.PixelFormat.Format32bppArgb)
+        public Bitmap(int width, int height) : this(width, height, PixelFormat.Format32bppArgb)
         {
         }
 
-        /// <include file='doc\Bitmap.uex' path='docs/doc[@for="Bitmap.Bitmap8"]/*' />
-        /// <devdoc>
-        ///    Initializes a new instance of the
-        /// <see cref='System.Drawing.Bitmap'/> 
-        /// class with the specified size and target <see cref='System.Drawing.Graphics'/>.
-        /// </devdoc>
         public Bitmap(int width, int height, Graphics g)
         {
             if (g == null)
-                throw new ArgumentNullException(SR.Format(SR.InvalidArgument, "g", "null"));
+            {
+                throw new ArgumentNullException(SR.Format(SR.InvalidArgument, nameof(g), "null"));
```

Comment:
All these don't need to use a custom string, it can just be `new ANE(nameof(g))`